stylesheets/layout/header.css

```diff
@@ -0,0 +1,77 @@
+.header-container {
+  background-color: var(--secondary-400);
+  padding: 1rem 3.2rem;
+  justify-content: space-between;
+}
+
+.header-logo {
+  font-weight: var(--font-w-medium);
+}
+
+.header-brand:hover {
+  color: var(--text-color);
+}
+
+.header-src {
+  max-width: 100%;
+  display: flex;
+}
+
+.header-src-input {
+  width: 100%;
+  border: 3px solid var(--secondary-hover);
+  padding: 5px;
+  height: 36px;
+  border-radius: 5px 0 0 5px;
+  outline: 2px solid transparent;
+  outline-offset: 2px;
+  color: var(--secondary-400);
+  font-size: var(--font-s-base);
+  font-weight: var(--font-w-medium);
+  transition: all 0.3s ease-in;
+}
+
+.header-src-input:focus {
+  color: var(--secondary-color);
+}
+
+.header-src-btn {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 40px;
+  height: 36px;

```

Comment:
Try making use of padding instead of providing fixed height and width.
Min-width/Max-Width and Max-height would be more applicable for these.

Reply:
Refactored